source/features/pagination-hotkey.tsx

```diff
@@ -20,9 +20,13 @@ const previousPageButtonSelectors = [
 	'.prh-commit > .BtnGroup > :first-child', // PR Commits
 ];
 
-function init(): void {
-	addHotkey(select(nextPageButtonSelectors), 'ArrowRight');
-	addHotkey(select(previousPageButtonSelectors), 'ArrowLeft');
+function init(signal: AbortSignal): void {
+	observe(nextPageButtonSelectors, button => {
+		addHotkey(button as HTMLButtonElement | HTMLAnchorElement, 'ArrowRight');
```

Comment:
I think you can add the actual tag name to the arrays _and_ append `as const`. e.g.

```ts
const previousPageButtonSelectors = [
	'a.previous_page', // Issue/PR list, Search
	'.paginate-container.BtnGroup > a:first-child', // Notifications
	'.paginate-container > .BtnGroup > a:first-child', // Commits
	'.paginate-container > .pagination > a:first-child', // Releases
 	'.prh-commit > .BtnGroup > a:first-child', // PR Commits
 ] as const
```

Double check the tag names on each page though

Reply:
I did that Typescript still fails